### PR TITLE
Remove reported reaction if it isn't the bot who added it.

### PR DESCRIPTION
**Pull request by Arzte**
smh.

### CODE REVIEW

File: src/megabot-internals/inquirer.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ module.exports = {
               if (ids.emojis.reported) msg.addReaction(`${ids.emojis.reported.name}:${ids.emojis.reported.id}`)
               queue.createDeletionRequest(await zd.getSubmission(question.zd_id, ['users', 'topics']))
             }
-          }
+          } else if (emoji.id === ids.emoji.reported.id) return msg.removeReaction(`${ids.emojis.report.name}:${ids.emojis.reported.id}`, userID) // no one but us should add this emoji :angery:
```
Comment (Dougley) [inline src/megabot-internals/inquirer.js]: Please put this at the top of the block.

Comment (Arzte) [reply src/megabot-internals/inquirer.js]: What, don't you like wasting time checking every other type of reaction?

Comment (Dougley) [reply src/megabot-internals/inquirer.js]: While you're at it, the reported emoji isn't a guaranteed object in contrary to the other emotes
```suggestion
          } else if (ids,emojis.reported && emoji.id === ids.emoji.reported.id) return msg.removeReaction(`${ids.emojis.report.name}:${ids.emojis.reported.id}`, userID) // no one but us should add this emoji :angery:
```

File: src/megabot-internals/inquirer.js [hunk 2]
```diff
@@ -97,7 +97,8 @@ module.exports = {
 
       switch (question.type) {
         case 1: { // feed vote
-          if (emoji === ids.emojis.upvote.id || emoji.id === ids.emojis.downvote.id) zd.applyVote(userID, question.zd_id, (emoji.id === ids.emojis.upvote.id) ? 'up' : 'down')
+          if (emoji.id === ids.emoji.reported.id) return msg.removeReaction(`${ids.emojis.report.name}:${ids.emojis.reported.id}`, userID) // no one but us should add this emoji :angery:
+          else if (emoji === ids.emojis.upvote.id || emoji.id === ids.emojis.downvote.id) zd.applyVote(userID, question.zd_id, (emoji.id === ids.emojis.upvote.id) ? 'up' : 'down')
```
Comment (Dougley) [inline src/megabot-internals/inquirer.js]: Please use `else if` as little as possible, granted I used it unnecessarily too.
```suggestion
          if (emoji === ids.emojis.upvote.id || emoji.id === ids.emojis.downvote.id) zd.applyVote(userID, question.zd_id, (emoji.id === ids.emojis.upvote.id) ? 'up' : 'down')
```

**Pull request merged**